test/jasmine/tests/gl_plot_interact_test.js

```diff
@@ -54,10 +52,9 @@ describe('Test gl plot interactions', function() {
         destroyGraphDiv();
     });
 
+    // ...
     function delay(done) {
-        setTimeout(function() {
-            done();
-        }, PLOT_DELAY);
+        setTimeout(done, 0);
```

Comment:
We need a 0 second timeout for the gl3d hover labels to test to pass reliably (at least on my machine).

More info in: https://github.com/plotly/plotly.js/issues/122#issuecomment-188868269